parts/kubernetesmastercustomdata.yml

```diff
@@ -15,6 +15,7 @@ runcmd:
 - echo "deb {{{dockerEngineDownloadRepo}}} ubuntu-xenial main" | sudo tee /etc/apt/sources.list.d/docker.list
 - "echo \"Package: docker-engine\nPin: version {{{dockerEngineVersion}}}\nPin-Priority: 550\n\" > /etc/apt/preferences.d/docker.pref"
 - apt-get update
+- apt-get install -y ebtables
```

Comment:
We discussed and it doesn't look like there is a way to add a check here. The provision script can't contain apt commands due to race conditions, either.